demos/android/MASVS-PRIVACY/MASTG-DEMO-00de3/MASTG-DEMO-00de3.md

```diff
@@ -0,0 +1,36 @@
+---
+platform: android
+title: Determine if Sensitive Data are sent to Firebase Analytics with Frida
+id: MASTG-DEMO-00de3
+code: [kotlin]
+test: MASTG-TEST-02te3
+---
+
+## Sample
+
+This sample demonstrates an Android application that sends sensitive user information to Firebase Analytics using the `logEvent` method. The app collects the user's blood type and user ID, which are considered sensitive data (health information), and transmits them to Firebase Analytics.
```

Comment:
Let's use the language from the Data Safety section. This is [data sharing](https://support.google.com/googleplay/android-developer/answer/10787469?hl=en#sharing&zippy=%2Cdata-sharing):

> From your app libraries and SDKs. Transferring data collected from your app off a user’s device directly to a third party via libraries and/or SDKs included in your app.

```suggestion
This sample collects the following [user data](https://support.google.com/googleplay/android-developer/answer/10787469?hl=en#types&zippy=%2Cdata-types) and sends it to Firebase Analytics using the `logEvent` method:

- User ID (**Data type:** User IDs, **Category:** Personal info)
- Blood type (**Data type:** Health info, **Category:** Health and fitness)
```